docs/storagenetwork/storagenetwork.md

```diff
@@ -0,0 +1,239 @@
+---
+sidebar_position: 1
+sidebar_label: Storage Network
+title: ""
+---
+
+# Storage Network
+
+Harvester uses Longhorn as the default storage plugin to provide VM and pod volumes. In case users expect to isolate Longhorn replication traffic from the Kubernetes cluster network or other network workloads. Users can assign a dedicated network for Longhorn replication traffic for dedicated network bandwidth.
+
+For more informations, please refer [Longhorn Storage Network](https://longhorn.io/docs/1.3.2/advanced-resources/deploy/storage-network/)
+
+:::note
+
+We always recommend using Harvester Storage Network setting instead of Longhorn Storage Network setting directly. Manual configuration in the Longhorn setting might confuse the determinations of the Harvester controller.
+
+:::
+
+## Prerequisites
+
+There are some prerequisites before configuring the Harvester Storage Network setting.
+
+- Well-configured Cluster Network and VLAN Config
+    - Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes. 
+- All VMs should be stopped.
+    - We recommend checking the VM status with the following command and should get an empty result.
+    - `kubectl get -A vmi`
+- All pods that are attached to Longhorn Volumes should be stopped.
+    - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
+
+## Configuration Example
+
+- IP range for Storage Network
+	- IP range should not conflict or overlap with Kubernetes cluster networks.
+	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
+	- If your cluster has 250 nodes, the IP range should be larger than /22.
+- Well-configured Cluster Network and VLAN Config
+	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
+- VLAN ID
+	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
+
+We will take the following configuration as an example to explain the details of the Storage Network
+
+- IP range: `192.168.0.0/24`
+- Cluster Network: `storage`
+- VLAN ID for Storage Network: `100`
```

Comment:
```suggestion
- VLAN ID
	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
- Well-configured Cluster Network and VLAN Config
	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
- IP range for Storage Network
	- IP range should not conflict or overlap with Kubernetes cluster networks(both `10.42.0.0/16` and `10.43.0.0/16` are reserved).
	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node, and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
	- If your cluster has 250 nodes, the IP range should be larger than `/22`.


We will take the following configuration as an example to explain the details of the Storage Network

- VLAN ID for Storage Network: `100`
- Cluster Network: `storage`
- IP range: `192.168.0.0/24`
```